scriptmodules/ports/jfduke3d.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="jfduke3d"
+rp_module_desc="lightweight Duke3D source port by JonoF"
+rp_module_licence="GPL2 https://raw.githubusercontent.com/jonof/jfduke3d/master/GPL.TXT"
+rp_module_repo="git https://github.com/jonof/jfduke3d.git master"
+rp_module_section="exp"
+rp_module_flags=""
+
+function depends_jfduke3d() {
+    local depends=(
+        libsdl2-dev libvorbis-dev libfluidsynth-dev
+    )
+
+    isPlatform "x86" && depends+=(nasm)
+    isPlatform "gl" || isPlatform "mesa" && depends+=(libgl1-mesa-dev libglu1-mesa-dev)
+    isPlatform "x11" && depends+=(libgtk3.0-dev)
```

Comment:
There isn't a package libgtk3.0-dev - should it be libgtk-3-dev ?

Reply:
You're right, thanks for the correction.

Reply:
Just in case: I force pushed the fix to this; the script is referencing the correct package now (libgtk-3-dev).